app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md

```diff
@@ -9,43 +9,78 @@ description: |-
 
-- biology 
-- chemistry 
-- computing 
-- design and technology 
-- English
-- languages 
-- mathematics
-- physics 
-- religious education
+You’ll choose your SKE course provider, but your teacher training provider can help you decide on the right one for you.
 
-You can also do a SKE course in primary mathematics.
+### SKE course subjects
 
-SKE course lengths can range from 8 to 28 weeks. They're usually online, but some can be classroom based and some can be a mixture of both. You can choose the provider you want to complete your SKE course with.
+SKE courses are currently available in 9 secondary subjects:
 
-Courses are available across England. Most candidates complete a SKE course before starting their teacher training year, but it’s sometimes possible to complete it alongside your initial teacher training (ITT) course. You may be able to complete the SKE course on a part-time basis.
+* biology
+* chemistry
+* computing
+* design and technology
+* English
+* languages
+* mathematics
+* physics
+* religious education
 
-You may also be eligible for a bursary of £175 per week if you train to teach in the 2022/23 academic year. This will go towards any extra costs you may have during the SKE course. Contact your ITT and SKE providers for more information.  
+You can also do one in primary mathematics.
+
+## How do SKE courses work?
+
+They're usually online, but some can be classroom based and some can be a mixture of both. 
+
+Course lengths range from 8 to 28 weeks and are usually 25 hours per week. Your provider will decide which course length is appropriate, based on your gap in knowledge. Different providers may offer different course lengths.
+
+You can also do a part-time SKE course and complete it over a longer period, or sometimes shorter. Your SKE provider will be able to discuss your options with you.
+
+## When should you complete your course?
+
+You should complete your SKE course before starting your teacher training. If you run out of time, your provider may allow you to continue your SKE course at the same time as your teacher training.
+
+## Get a SKE bursary
+
+If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time SKE course and complete less than 25 hours a week, this will be paid in proportion to your course hours. How and when your bursary is paid will vary by your training provider.
+
+You can use your SKE bursary for any extra costs you may have during your SKE course, for example, making up for any lost income.
+
+### Eligibility for a SKE bursary
+
+You’ll need (one of the following):
+
+* a bachelor’s degree class 2:2 or above 
+* a master’s degree
+* a PhD
+
+You'll need to complete your SKE course before your teacher training starts to receive your SKE bursary.
+
+$international-content$
```

Comment:
@gazcarr @ethax-ross I've added the inset text for the international content (as per the ticket [#4053](https://trello.com/c/IQqvPIp5/4053-consider-design-solution-to-highlight-differences-for-international-teachers-and-trainees)) but it's coming up as the yellow highlighted indented text. Not sure how to change it to match your design Gary so it's indented but without the yellow background?